js/app/src/components/StatusTracker/ToastContent.svelte

```diff
@@ -0,0 +1,173 @@
+<script lang="ts">
+	import { Alert, Information, Warning } from "@gradio/icons";
+	import { createEventDispatcher, onMount, afterUpdate } from "svelte";
+	import { fade } from "svelte/transition";
+	import type { ThemeMode } from "../types";
+
+	export let message: string = "";
+	export let description: string = "";
+	export let type: string = "";
+	export let theme_color: string = "";
+	export let theme_mode: ThemeMode;
+	export let id: number;
+
+	const dispatch = createEventDispatcher();
+
+	function close_message() {
+		dispatch("close", id);
+	}
+
+	let lightStyles = {
+		"toast-bg-color": `var(--color-${theme_color}-50)`,
+		"toast-text-color": `var(--color-${theme_color}-700)`,
+		"toast-border-color": `var(--color-${theme_color}-700)`,
+		"toast-icon-color": `var(--color-${theme_color}-700)`
+	};
+
+	let darkStyles = {
+		"toast-bg-color": `var(--color-grey-950)`,
+		"toast-text-color": `var(--color-${theme_color}-50)`,
+		"toast-border-color": `var(--color-${theme_color}-500)`,
+		"toast-icon-color": `var(--color-${theme_color}-500)`
+	};
+
+	let cssVarStyles = "";
+
+	function updateCssVarStyles() {
+		cssVarStyles = Object.entries(
+			theme_mode === "light" ? lightStyles : darkStyles
+		)
+			.map(([key, value]) => `--${key}:${value}`)
+			.join(";");
+	}
+
+	afterUpdate(updateCssVarStyles);
+
+	onMount(() => {
+		setTimeout(() => {
+			close_message();
+		}, 10000);
+	});
+
+	$: updateCssVarStyles();
```

Comment:
This won't react to changes in props since there's no variable prop in this line. If you instead remove the function and directly set cssVarStyles above as
```
$: cssVarStyles = Object.entries(...
```
then it should react to changes in theme_mode